news/2024/2024-07-15-the-followpoint-whitecat-the-cat-who-dominated-osu.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "The Followpoint: WhiteCat, the cat who dominated osu!"
+date: 2024-07-15 17:00:00 +0000
+series: The Followpoint
+---
+
+Dive into an interview with WhiteCat, a legendary *cat* and an osu! player!
+
+![](/wiki/shared/news/2024-07-15-the-followpoint-whitecat-the-cat-who-dominated-osu/banner.jpg)
+
+Hey, did you hear that? That's the sound of a new interview! Today we are joined by WhiteCat, a top player who many remember as someone who was dominating the scene before mrekk's rise as a former rank #1 player!
+
+So, let's get right to reading!
+
+---
+
+*Hello, let's start with an introduction, how would you introduce yourself to people who don't know you?*
+
+> Hi, I'm WhiteCat, former nr.1 osu! player, now kinda retired.
+
+*How did you get introduced to osu! and what was your first impression of it?*
+
+> I got introduced to osu! back when MCPvP disbanded, yet I really did not like it the first time I tried it. I layed off the game for roughly a year and then came back and stuck with it 
+
+*I know many have already heard about it, but could you please tell us how did you come up with your username “WhiteCat”, for people who don’t know that?*
+
+> The story begins with a neighborhood black cat who used to visit us. Took inspiration from her and wanted to name myself BlackCat. As this was taken though, I opted for WhiteCat.
+
+*Since you have some experience with leading a healthy life, do you feel like there can be a big mental and physical help by visiting a gym?*
+
+> I think there are numerous benefits to the going to the gym. I think this applies to a lot of types of exercise though, and in the end it ties back to moving your body a lot
+
+*What was the most memorable moment for you in your osu! career?*
+
+> Most definitely the unban (WAS?)
```

Comment:
will need @peppy's opinion on this one as it's about restrictions

Reply:
A lil update, due to style change it's now line 108

Reply:
removing these instead in order to prevent delays in the schedule down the line